src/main/java/me/botsko/prism/actions/BlockChangeAction.java

```diff
@@ -48,12 +49,33 @@
      */
     @Override
     public String getNiceName() {
+    	if (actionData == null) actionData = getActionData();
         String name = "";
+        if( actionData instanceof SkullActionData ) {
```

Comment:
Thanks for adding these; however:

1. Could this be moved to a separate commit, please?
2. Is this duplicating a lot of code from `getNiceName` in `BlockAction`? If so, could these be moved to a common utility method instead?